ldk/node/src/hostClients/whisperMessageBuilder.ts

```diff
@@ -230,6 +230,27 @@ export const generateWhisperListElement = (
       msg.setDivider(inputMsg);
       break;
     }
+    case 'link': {
+      const inputMsg = new WLE.Link();
+      if (element.align) {
+        inputMsg.setAlign(generateWhisperListAlign(element.align));
+      } else {
+        inputMsg.setAlign(generateWhisperListAlign(WhisperListAlign.LEFT));
```

Comment:
Does it make sense consolidate this into the line above?

something like

```typescript
inputMsg.setAlign(generateWhisperListAlign(element.align || WhisperListAlign.LEFT));
```

Reply:
Made that change